samples/posix/uname/src/main.c

```diff
@@ -0,0 +1,186 @@
+/*
```

Comment:
This is great - we may actually want to move this into `lib/posix` at some point. Thanks for adding it :-)